documentation/modules/ipi-install-preparing-the-provision-node-for-openshift-install.adoc

```diff
@@ -84,6 +84,32 @@ For more information about Red Hat Subscription Manager, see link:https://access
 This step can also be run from the web console.
 ====
 +
+ifeval::[{release} >= 4.4]
+----
+
+[kni@provisioner ~]$ export PUB_CONN=<baremetal_nic_name>
+[kni@provisioner ~]$ export PROV_CONN=<prov_nic_name>
+[kni@provisioner ~]$ sudo nohup bash -c '
+    nmcli con down "$PROV_CONN"
+    nmcli con down "$PUB_CONN"
+    nmcli con delete "$PROV_CONN"
+    nmcli con delete "$PUB_CONN"
+    # RHEL 8.1 appends the word "System" in front of the connection, delete in case it exists
+    nmcli con down "System $PUB_CONN"
+    nmcli con delete "System $PUB_CONN"
+    nmcli connection add ifname provisioning type bridge con-name provisioning
+    nmcli con add type bridge-slave ifname "$PROV_CONN" master provisioning
+    nmcli connection add ifname baremetal type bridge con-name baremetal
+    nmcli con add type bridge-slave ifname "$PUB_CONN" master baremetal
+    nmcli con down "$PUB_CONN";pkill dhclient;dhclient baremetal
+    nmcli connection modify provisioning ipv6.addresses fd00:1101::1/64 ipv6.method manual
```

Comment:
line 165 is the one that changed here compared to what it was previous and id like to evaluate the diff bc OCP 4.3 will stick with IPv4 provisioning